### ISSUE TITLE
--recursive flag doesn't seem to run in child directories

### ISSUE THREAD

**Issue by caoimheharvey** [terraform-linters/tflint#2132]
### Summary

I have my `.tflint.hcl` config in a parent directory and run `tflint --init` in the parent directory, and I have a lot of child directories that I want to lint, but when I run `tflint --recursive --config="$(pwd)/.tflint.hcl"` from the parent directory it returns nothing even if there are linting failures. 
When I switch to debug output this is what I am seeing:
```
│ Failed to run in child/nested/nested/nested; exit status 1
Failed to initialize plugins; Plugin "terraform" not found. Did you run "tflint --init"?
│ Failed to run in child; exit status 1
Failed to initialize plugins; Plugin "terraform" not found. Did you run "tflint --init"?
│ Failed to run in child/nested; exit status 1
Failed to initialize plugins; Plugin "terraform" not found. Did you run "tflint --init"?
│ Failed to run in child/nested; exit status 1
```
Seems odd that I would need to run `tflint --init --recursive` because I would very quickly hit a rate limit (tried this), so its not a practical solution. 
As a work-around I've tried to copy the `.tflint.d` directory to all my subdirectories but again, this isn't a workable solution. 

### Command

tflint --recursive --config="$(pwd)/.tflint.hcl"

### Terraform Configuration

```terraform
Not a terraform specific issue as far as I can tell
```

### TFLint Configuration

```HCL
config {
  plugin_dir = "./.tflint.d/plugins"
}

plugin "terraform" {
  enabled = true
  version = "0.9.0"
  source  = "github.com/terraform-linters/tflint-ruleset-terraform"
}

plugin "aws" {
  enabled = true
  version = "0.33.0"
  source  = "github.com/terraform-linters/tflint-ruleset-aws"
}
```

### Output

```Shell
│ Failed to run in child/nested/nested/nested; exit status 1
Failed to initialize plugins; Plugin "terraform" not found. Did you run "tflint --init"?
│ Failed to run in child; exit status 1
Failed to initialize plugins; Plugin "terraform" not found. Did you run "tflint --init"?
│ Failed to run in child/nested; exit status 1
Failed to initialize plugins; Plugin "terraform" not found. Did you run "tflint --init"?
│ Failed to run in child/nested; exit status 1
```
```


### TFLint Version

0.53.0

### Terraform Version

1.9.5

### Operating System

- [X] Linux
- [X] macOS
- [ ] Windows
